components/http_foundation/trusting_proxies.rst

```diff
@@ -17,7 +17,7 @@ the actual host may be stored in an ``X-Forwarded-Host`` header.
 
 Since HTTP headers can be spoofed, Symfony does *not* trust these proxy
 headers by default. If you are behind a proxy, you should manually whitelist
-your proxy.
+your proxy as follows:
```

Comment:
Should we move the `versionadded` block somewhere as it may not read a bit weird with the colon here?